README.md

```diff
@@ -3,8 +3,6 @@ available on the website.
 
 # Development
 
```

Comment:
```suggestion

**Note: The site is currently built with Zola version 0.19.2.**

```
Although given you ignored this same suggestion in #63, I'm presuming there's some reason this is bad.

I would however like to be told that